js-exercises/telephone-check/telephoneCheck.test.js

```diff
@@ -0,0 +1,18 @@
+import { telephoneCheck } from "./telephoneCheck";
+
+describe("telephoneCheck", () => {
+  test("should return correct boolean", () => {
```

Comment:
Both regexes fail on this input: `1  (555) 555-5555`. 
Can you take this into account?

Reply:
This is the first check in test suit. All, following tests are passing for me. Can you please confirm this `1 (555) 555-5555` again?